Looms.PoS/Looms.PoS.Application/Features/Payment/Commands/CreatePayment/CreatePaymentRequestValidator.cs

```diff
@@ -0,0 +1,39 @@
+﻿using FluentValidation;
+using Looms.PoS.Application.Models.Requests;
+using Looms.PoS.Domain.Enums;
+
+namespace Looms.PoS.Application.Features.Payment.Commands.CreatePayment;
+
+public class CreatePaymentRequestValidator : AbstractValidator<CreatePaymentRequest>
+{
+    public CreatePaymentRequestValidator()
+    {
+        RuleFor(x => x.OrderId)
+            .NotEqual(Guid.Empty);
```

Comment:
Validate that it is valid guid

Reply:
Don't need as OrderId in CreatePaymentRequest is type of Guid. If invalid Guid is sent, json conversion fails. I can only change it to string, validate and then convert to Guid.